modules/ROOT/partials/misc/plugin-menu-item-id-boilerplate.adoc

```diff
@@ -1,28 +1,29 @@
-{% if page.name == "available-menu-items.md" %}
+ifeval::["{docname}" == "advanced/available-menu-items"]
 == {pluginname} plugin
 
 The {pluginname} plugin provides the following menu items:
 
-{% if altplugincode %}
+ifdef::altplugincode[]
 include::partial$menu-item-ids/{altplugincode}-menu-items.adoc[]
-{% else %}
+endif::altplugincode[]
+ifndef::altplugincode[]
 include::partial$menu-item-ids/{plugincode}-menu-items.adoc[]
-{% endif %}
+endif::altplugincode[]
 
-For information on the {pluginname} plugin, see: link:{baseurl}/plugins/{plugincategory}/{plugincode}/[Plugins - The {pluginname} plugin]
-
-{% else %}
+For information on the {pluginname} plugin, see: xref:plugins/{plugincategory}/{plugincode}.adoc[Plugins - The {pluginname} plugin]
+endif::[]
+ifeval::["{docname}" != "advanced/available-menu-items"]
 == Menu items
 
 The {pluginname} plugin provides the following menu items:
 
-{% if altplugincode %}
+ifdef::altplugincode[]
 include::partial$menu-item-ids/{altplugincode}-menu-items.adoc[]
-{% else %}
+endif::altplugincode[]
+ifndef::altplugincode[]
 include::partial$menu-item-ids/{plugincode}-menu-items.adoc[]
-{% endif %}
+endif::altplugincode[]
 
 include::partial$misc/menu-item-settings.adoc[]
-
-{% endif %}
-:altplugincode: nil
+endif::[]
+:!altplugincode:
```

Comment:
This is a poor practice and something we should remove. As a general rule we should only unset an attribute if we set it in the same file. The reason being that this might need to be used elsewhere in the file that includes this and unsetting this at the end would break it.

I'd suggest looking at how this is done in the TinyMCE 6 branch as this file should be almost identical.

Reply:
Ok, I'll change the logic to make it similar to TinyMCE 6 branch

Reply:
ok, I changed it with [this commit](https://github.com/tinymce/tinymce-docs/pull/2272/commits/5b3a19bf00023af415537b2b065876e7d2ad8776), I also changed: `endif::altplugincode[]` to `endif::[]` it is less verbose and works in the same way.

I had changed a couple of files related to plugins because those files used `altplugincode`, if it is too much out of scope I'll change it back